Hakawai/Mentions/HKWMentionsCustomChooserViewDelegate.h

```diff
@@ -0,0 +1,28 @@
+/*!
+ This is being used for consumer of the library that wants to provide its own custom chooser view.
+ The consumer is responsible for data processing, cool down logic and UI setup.
+ */
+@protocol HKWMentionsCustomChooserViewDelegate <NSObject>
+
+/*!
+ Notify the delegate that key string for the mentions query has been updated. Delegate then can fetch the mentions results. Upon
+ completion, it will call `dataReturnedWithEmptyResults:keystringEndsWithWhiteSpace:` to notify the mentions plugin.
+
+ \param keyString          the string that the data source should search upon
+ \param character          if \c type is Explicit, this is the control character that was typed to begin the
+                           mention creation; otherwise, it should be ignored
+ */
+- (void)didUpdateKeyString:(nonnull NSString *)keyString
```

Comment:
are these actually implemented/used? or is this just a startin rb

Reply:
just starting the rb